scipy/stats/_levy_stable_distn.py

```diff
@@ -572,22 +572,23 @@ def _fitstart_S1(data):
     psi_2 = interpolate.interp2d(
         nu_beta_range, nu_alpha_range, beta_table, kind="linear"
     )
-    psi_2_1 = lambda nu_beta, nu_alpha: (
-        psi_2(nu_beta, nu_alpha) if nu_beta > 0 else -psi_2(-nu_beta, nu_alpha)
-    )
+
+    def psi_2_1(nu_beta, nu_alpha):
+        return psi_2(nu_beta, nu_alpha) if nu_beta > 0 else -psi_2(-nu_beta, nu_alpha)
```

Comment:
Haha. Wow, I should really pay attention better. I thought I was being clever by doing a regex find on this block and and didn't even check that the result was sensible. Sorry about that.

Reply:
haha that's okay! I actually went and initially checked if you had just used some esoteric syntax I'd never seen.